src/modules/sensors/vehicle_optical_flow/VehicleOpticalFlow.cpp

```diff
@@ -358,7 +358,7 @@ void VehicleOpticalFlow::UpdateDistanceSensor()
 
 			if (_distance_sensor_subs[i].update(&distance_sensor)) {
 				// only use the first instace which has the correct orientation
-				if ((hrt_elapsed_time(&distance_sensor.timestamp) < 100_ms)
+				if ((hrt_elapsed_time(&distance_sensor.timestamp_sample) < 100_ms)
```

Comment:
For this simple alive check probably use `timestamp` for the same reason as above?

```suggestion
				if ((hrt_elapsed_time(&distance_sensor.timestamp) < 100_ms)
```